ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/HomePage.kt

```diff
@@ -29,30 +29,26 @@ import com.microsoft.device.dualscreen.windowstate.WindowState
 
 @Composable
 fun ComposeGalleryApp(windowState: WindowState) {
-    // Check if app should be in dual mode
-    val isDualMode = windowState.isDualPortrait()
-
     // Get relevant image data for the panes
     val models = DataProvider.imageModels
 
     // Remember app state variables
     var selectedImageIndex by rememberSaveable { mutableStateOf(0) }
     val updateImageIndex: (Int) -> Unit = { index -> selectedImageIndex = index }
 
-    ComposeGalleryAppContent(models, isDualMode, selectedImageIndex, updateImageIndex)
+    ComposeGalleryAppContent(models, selectedImageIndex, updateImageIndex)
 }
 
 @Composable
 fun ComposeGalleryAppContent(
     models: List<ImageModel>,
-    isDualMode: Boolean,
     selectedImageIndex: Int,
     updateImageIndex: (Int) -> Unit,
 ) {
     TwoPaneLayout(
         paneMode = TwoPaneMode.HorizontalSingle,
-        pane1 = { ListPane(models, isDualMode, selectedImageIndex, updateImageIndex) },
-        pane2 = { DetailPane(models, isDualMode, selectedImageIndex) }
+        pane1 = { ListPane(models, !isSinglePane, selectedImageIndex, updateImageIndex) },
```

Comment:
`isSinglePane` is not necessary here either. We can just call `isSinglePane` inside `TwoPaneScope.ListPane` directly.